index.html

```diff
@@ -2001,9 +2001,29 @@
       <section>
         <h3>Frequency of Claim Issuance</h3>
 
-        <p class="issue">
-The rate at which an issuer issues claims may be a privacy violation.
+        <p>
+As Section <a href="#usage-patterns"></a> details, usage patterns can be
```

Comment:
Is the use of an empty link here correct? Does ReSpec do the right thing? Is `Section` meant to be capitalized?

Reply:
Yes, yes, and yes.